### PR TITLE
Add a new workflow to publish docker images

### PR DESCRIPTION
**Pull request by supl**
This PR creates a GitHub Actions workflow to release the `ghcr.io/scalar-labs/scalar-admin` image.

### CODE REVIEW

File: .github/workflows/docker.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.CR_PAT }}
+
+      - run: ./gradlew docker
+
+      - name: Run Trivy vulnerability scanner
```
Comment (feeblefakie) [inline .github/workflows/docker.yml:32]: Should it be checked before creating a tag?

Comment (supl) [reply .github/workflows/docker.yml:34]: Ah, sorry, maybe doing scanning after tagging is better I guess?
I changed the order in https://github.com/scalar-labs/scalar-admin/pull/26/commits/fb90cd172a06795158ee6dcd5fde2348a01c33ee

**Review by kota2and3kan — APPROVED** [review]
LGTM! Thank!

I have a question that is not related to this PR directly.
Regarding Trivy, is it not necessary to check vulnerabilities in each commit/PR like other product (Scalar DB/DL)?

File: .github/workflows/docker.yml (hunk 1 above)
Comment (feeblefakie) [reply .github/workflows/docker.yml:34]: I meant the tagging that triggers this workflow.
If trivy fails, you need to remove the tag and retag it, which is a bit bothering.
So, I thought trivy can be checked before tagging.
But, it is probably difficult with GitHub Actions.

**Review by feeblefakie — APPROVED** [review]
LGTM! Thank you!

**Pull request merged**